__tests__/__datasets__/schema-deprecated.json

```diff
@@ -222,11 +222,9 @@
                 "deprecated": true,
                 "allOf": [
                   {
-                    "title": "option 1",
```

Comment:
Removed these because they were always going to be merged into a single `title` property.